app/forms/steps/screener/email_consent_form.rb

```diff
@@ -0,0 +1,24 @@
+module Steps
+  module Screener
+    class EmailConsentForm < BaseForm
+      include SingleQuestionForm
+      include HasOneAssociationForm
+
+      has_one_association   :screener_answers
+      yes_no_attribute      :email_consent
+      attribute             :email_address, String, reset_when_no: [:email_consent]
```

Comment:
The `reset_when_no` goes in the `yes_no_attribute` declaration. And we want to reset the email field, so it would be like this:

```ruby
yes_no_attribute :email_consent, reset_when_no: [:email_address]
attribute :email_address, NormalisedEmail
```